src/wp-admin/includes/image.php

```diff
@@ -1280,8 +1281,9 @@ function _copy_image_file( $attachment_id ) {
  */
 function wp_upload_image_mime_transforms( $attachment_id ) {
 	$image_mime_transforms = array(
-		'image/jpeg' => array( 'image/jpeg', 'image/webp' ),
-		'image/webp' => array( 'image/webp', 'image/jpeg' ),
+		'image/jpeg'      => array( 'image/jpeg', 'image/webp' ),
+		'image/webp'      => array( 'image/webp', 'image/jpeg' ),
+		'application/pdf' => array( 'image/jpeg' ),
```

Comment:
This ensures that the primary type used for PDF's when make_subsizes is called is jpeg, matching current core behavior. See https://github.com/WordPress/wordpress-develop/blob/7fc225aa87c80680df01a0a82e6bb94b7773a34c/src/wp-admin/includes/image.php#L627 for the existing call.

Reply:
This is still valid, I believe we should remove this in favor of relying on the `$file`'s MIME type rather than `get_post_mime_type()`. Probably in your testing this was failing because of relying on the latter.

Reply:
yep, I think that was it